### PR TITLE
shorten message & put on 2 lines

### LINKED ISSUE #2760: fix home tab Remi and twitter, medium icons
They are overlapping on each other

### PR DESCRIPTION
**Pull request by ryestew**
fixes #2760

### CODE REVIEW

File: libs/remix-ui/home-tab/src/lib/remix-ui-home-tab.tsx [hunk 1]
```diff
@@ -234,12 +234,18 @@ export const RemixUiHomeTab = (props: RemixUiHomeTabProps) => {
...
-            <i className="pl-4 text-danger fas fa-exclamation-triangle"></i>
-            <span className="px-2 remixui_home_text text-danger mt-4 pt-4">
-              Scam Alert: There are video tutorials going around that provide urls other than remix.ethereum.org, and could be scams. Also, beware of online videos promoting "liquidity front runner bots".
+          <div className="d-flex flex-column">
+            <span className="pl-4 text-danger mt-2">
+              <i className="pr-2 text-danger fas fa-exclamation-triangle"></i>
+              Scam Alerts:
+            </span>
+            <span className="pl-4 text-danger mt-1">
+              The only domain Remix uses is ethereum.org
```
Comment (Aniket-Engg) [inline libs/remix-ui/home-tab/src/lib/remix-ui-home-tab.tsx:243]: I think it should be `remix.ethereum.org`

Comment (LianaHus) [reply libs/remix-ui/home-tab/src/lib/remix-ui-home-tab.tsx:243]: is it domain? domain is only the second part, no?

Comment (ryestew) [reply libs/remix-ui/home-tab/src/lib/remix-ui-home-tab.tsx:243]: Also if we put in remix.ethereum.org then we need to put in remix-alpha.ethereum.org and remix-beta.ethereum.org.

File: libs/remix-ui/home-tab/src/lib/remix-ui-home-tab.tsx [hunk 2]
```diff
@@ -220,12 +220,30 @@ export const RemixUiHomeTab = (props: RemixUiHomeTabProps) => {
...
+                </span>
+                <span className="pl-4 text-danger mt-1">
+                  The only URL Remix uses is remix.ethereum.org 
+                </span>
+                <span className="pl-4 text-danger mt-1">        
+                  Beware of online videos promoting "liquidity front runner bots":  
+                  <a className="pl-2 remixui_home_text" target="__blank" href="https://medium.com/remix-ide/remix-in-youtube-crypto-scams-71c338da32d">Learn more</a>
+                </span>
+                <span className="pl-4 text-danger mt-1">
+                  Additional saftely tips: &nbsp;<a className="remixui_home_text" target="__blank" href="https://remix-ide.readthedocs.io/en/latest/security.html">here</a>
```
Comment (Aniket-Engg) [inline libs/remix-ui/home-tab/src/lib/remix-ui-home-tab.tsx:241]: typo. it should be `safety`

**Review by Aniket-Engg — APPROVED** [review]

**Pull request merged**